autoload/ctrlp/funky/ft/crystal.vim

```diff
@@ -0,0 +1,64 @@
+" Language: Crystal (crystal)
+" Author: phongnh <nhphong1406@gmail.com>
+" License: The MIT License
+
+let s:fu = ctrlp#funky#getutils()
```

Comment:
This variable looks like not used in the script. Can you please remove this?

Reply:
Removed. Please help to re-check this, thanks.